pkg/interactive/interactive.go

```diff
@@ -46,7 +72,7 @@ func GetString(input Input) (a string, err error) {
 		dflt = ""
 	}
 	question := input.Question
-	if !input.Required && dflt == "" {
+	if !input.Required && dflt == "" && !strings.Contains(strings.ToLower(question), "optional") {
```

Comment:
This looks like something that should be fixed in the parameter description, not here (and below). That's what the `required` attribute is for.

Reply:
@tbrisker unfortunately we are caught between the OCM UI and the ROSA Cli. We have the optional and required in the string of the description. So in ROSA CLI it's included twice. If we remove it from the addon parameter configuration we lose it in the UI.

Reply:
Actually I think this might be fixable from our side. I've opened an MR [1] so will remove the code here.
[1] https://gitlab.cee.redhat.com/service/managed-tenants/-/merge_requests/3036

Reply:
After merge of the above this is now working as expected so removed the code.

Reply:
🥳 glad you figured out a better way :)